test/quantization/test_quantize_fx.py

```diff
@@ -757,3 +768,177 @@ def forward(self, x):
             quantized,
             expected_node_occurrence=count_check,
             expected_node_list=order_check)
+
+class TestQuantizeFxModels(QuantizationTestCase):
+    @skipIfNoTorchVision
+    def test_torchvision(self):
+        from torchvision import models
+        from torchvision.models import quantization as quantized_models
+
+        def get_available_classification_models(models):
+            return [k for k, v in models.__dict__.items() if callable(v) and k[0].lower() == k[0] and k[0] != "_"]
+
+        model_list = get_available_classification_models(models)
+        quantized_model_list = get_available_classification_models(quantized_models)
+
+        no_pretrained_model = set(['shufflenet_v2_x0_5', 'shufflenet_v2_x1_5', 'shufflenet_v2_x2_0'])
+        quantized_model_list = set(quantized_model_list) - no_pretrained_model
+        # test eager and graph consistency
+        model_list = quantized_model_list
+        # slice need to be fixed in symbolic tracing(https://github.com/pytorch/pytorch/issues/43511)
+        model_list = set(model_list) - {'googlenet', 'inception_v3'}
+        # getattr should not be used as node name(https://github.com/pytorch/pytorch/issues/43522)
+        model_list -= {'shufflenet_v2_x1_0', 'mobilenet_v2'}
+
+        # mobilenet: dropout error RuntimeError: "bernoulli_scalar_cpu_" not implemented for 'QUInt8'
+        # incpetion_v3: looks like there is some problem with AuxLogits
+        quantized_not_working = [('qat', 'mobilenet_v2'),
```

Comment:
mobilenet_v2 should work with QAT. Why is this excluded here?

Reply:
not sure about this, currently mobilenet_v2 can't traced anyway, will revisit when it is enabled.

Reply:
the error I saw previously is: RuntimeError: "bernoulli_scalar_cpu_" not implemented for 'QUInt8'

Reply:
That corresponds to drop out being called with quantized tensors, which is incorrect. Does this mean that we are missing a dequantize somehere?

Reply:
dropout can support quantized tensor right? I think it is probably we set training to true and it is not supported. It might be fixed now actually since currently I always make model eval before convert